source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc

```diff
@@ -0,0 +1,20 @@
+[[Configuring_timezones]]
+= Configuring the time zone
+
+{virt-product-fullname} stores time zone configurations for virtual machines in */etc/ovirt-engine/conf/00-timezone.properties*. This file contains default time zone values such as `Etc/GMT=Greenwich Standard Time`. It features mappings that are valid for windows and non-Windows time zones.
+
+[IMPORTANT]
+====
+Do not edit the actual *00-timezone.properties* file. Changes will be overwritten if you upgrade or restore the {engine-name}.
+
+Do not change values that come directly from the operating system or the {engine-name}.
+====
+
```

Comment:
Add  .Procedure and begin line 13 ". Create an override ..."

Reply:
Fixed.